### PR TITLE
Update fields.yml in azure module

### PR DESCRIPTION
**Pull request by narph**
## What does this PR do?

Update fields.yml in azure module

## Why is it important?

Update fields.yml in azure module

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: x-pack/metricbeat/module/azure/data.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func EventsMapping(metrics []Metric, metricset string, report mb.ReporterV2) err
 				event, metricList = createEvent(timestamp, defaultMetric, groupTimeValues)
 			}
 			if metricset == nativeMetricset {
-				event.ModuleFields.Put("metrics", metricList)
+				event.ModuleFields.Put(fmt.Sprintf("%s.metrics", nativeMetricset), metricList)
```
Comment (jsoriano) [inline x-pack/metricbeat/module/azure/data.go:87]: Would this work the same?
```suggestion
				event.MetricSetFields.Put("metrics", metricList)
```

**Review by jsoriano — APPROVED** [review]
LGTM, a small suggestion but feel free to ignore it.

**Review by jsoriano — APPROVED** [review]
:+1:

File: x-pack/metricbeat/module/azure/data.go (hunk 1 above)
Comment (narph) [reply x-pack/metricbeat/module/azure/data.go]: reverted change

**Review by jsoriano — APPROVED** [review]

**Pull request merged**